integration/src/test/java/org/mule/test/routing/AsyncTestCase.java

```diff
@@ -115,14 +119,14 @@ public void withMaxConcurrencyAsyncDispatched() throws Exception {
   }
 
   @Test
-  @Story(BACKPRESSURE)
+  @Stories({@Story(BACKPRESSURE), @Story(ExecutionEngineStory.MAX_CONCURRENCY)})
   @Description("Assert that if no maxConcurrency is configured for an async, the value from the flow is inherited")
   public void withFlowMaxConcurrency() throws Exception {
     testAsyncMaxConcurrency("with-flow-max-concurrency");
   }
 
   @Test
-  @Story(BACKPRESSURE)
+  @Stories({@Story(BACKPRESSURE), @Story(ExecutionEngineStory.MAX_CONCURRENCY)})
```

Comment:
Static import.

Reply:
cannot, there's already a constant with that identifier